### PR TITLE
#156004609 Pass data from main activity to detail activity

### PR DESCRIPTION
**Pull request by ongebo**
### What does this PR do?
Passes data from main activity to detail activity using **Intent Extras** to display details for a single profile
#### Description of the task to be completed?
- Update activity_detail.xml layout file to use values in strings.xml
- Remove redundant Button view in the activity_main.xml layout file
- Update showProfileDetails() method in MainActivity.java
- Update DetailActivity.java to show details for a single profile
#### How should this be manually tested?
- Run the project in Android Studio on this branch (Ensure there is an Internet Connection)
- Click on a single profile on the home page
- Verify that a details page for that specific profile is displayed
#### Screenshots
pass data
#### What are the relevant Pivotal Tracker stories?
[#156004609](https://www.pivotaltracker.com/story/show/156004609)

### CODE REVIEW

File: app/src/main/java/com/example/javacodersnairobi/adapter/GithubAdapter.java [hunk 1]
```diff
@@ -9,20 +9,21 @@
...
 
 import java.util.List;
 
-public class GithubAdapter extends RecyclerView.Adapter<GithubAdapter.GithubUsersViewHolder> {
+public class GithubAdapter extends RecyclerView.Adapter<GithubAdapter.MyViewHolder> {
     private List<GithubUsers> dataSet;
     private Context context;
 
-    public static class GithubUsersViewHolder extends RecyclerView.ViewHolder {
+    public static class MyViewHolder extends RecyclerView.ViewHolder {
```
Comment (damafeez) [inline app/src/main/java/com/example/javacodersnairobi/adapter/GithubAdapter.java]: Why did you have to change this class' name to "MyViewHolder", I feel it isn't descriptive enough. Isn't the old name just fine? "GithubUsersViewHolder" or "UsersViewHolder". Please find a more suitable name.

**Review by damafeez — CHANGES_REQUESTED** [review]
Very good work @ongebo. Please look into the comment I left you. Otherwise, this looks really good to me.

File: app/src/main/java/com/example/javacodersnairobi/adapter/GithubAdapter.java (hunk 1 above)
Comment (ongebo) [reply app/src/main/java/com/example/javacodersnairobi/adapter/GithubAdapter.java]: Thanks for this observation @damafeez, I have made the corresponding adjustment.

**Review by damafeez — APPROVED** [review]
LGTM

**Pull request merged**